Calendr/Components/Label.swift

```diff
@@ -79,3 +89,65 @@ class Label: NSTextField {
             .disposed(by: disposeBag)
     }
 }
+
+class CalendarLabel: NSTextField {
```

Comment:
I think we can inject an `Observable` in the `Label` constructor, with a default value of `Scaling.observable`.
That way we can achieve the same behavior without creating another component.

Also, since we're injecting `calendarTextScaling`, we can avoid creating the `CalendarTextScaling` singleton as well.

Reply:
Great👍
Feel free to change everything as you think fits the project best🙂